types/header.go

```diff
@@ -50,6 +66,8 @@ type Header struct {
 
 type BlockNumber U32
 
+type AppId UCompact
```

Comment:
Not sure if this is used in header properly! 
You could take this as [https://github.com/availproject/go-substrate-rpc-client/blob/7fc7f655e4ed45d5237cab61be218538a94d068c/types/header.go#L35](ref)